ReactiveCocoaTests/Swift/TestError.swift

```diff
@@ -14,3 +16,6 @@ enum TestError: Int {
 
 extension TestError: ErrorType {
 }
+
+class TestNSError: NSError {
```

Comment:
It's discouraged to subclass `NSError`. Instead, you should differentiate errors using the `domain`, like you're doing in the test.

Can you change the test to just use `NSError`?